x-pack/plugin/transform/qa/multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/continuous/TransformContinuousIT.java

```diff
@@ -276,7 +275,7 @@ public void testContinousEvents() throws Exception {
 
             // at random we added between 0 and 999_999ns == (1ms - 1ns) to every data point, so we add 1ms, so every data point is before
             // the checkpoint
-            waitUntilTransformsReachedUpperBound(runDate.toEpochMilli() + 1, run);
+            waitUntilTransformsProcessedNewData(ContinuousTestCase.SYNC_DELAY, run);
```

Comment:
Just double-checking: Is "+1" no longer needed here?

Reply:
I removed the outdated comment. The test waits according to `delay` with `greaterThan`, so it _should_ be fine.